frontend/rust-lib/flowy-database2/src/entities/type_option_entities/checkbox_entities.rs

```diff
@@ -1,24 +1,32 @@
 use crate::services::field::CheckboxTypeOption;
 use flowy_derive::ProtoBuf;
 
+#[derive(Default, Debug, Clone, ProtoBuf)]
+pub struct CheckboxCellDataPB {
+  #[pb(index = 1)]
+  pub is_check: bool,
+}
+
+impl CheckboxCellDataPB {
+  pub fn new(is_check: bool) -> Self {
+    Self { is_check }
+  }
+}
+
 #[derive(Debug, Clone, Default, ProtoBuf)]
 pub struct CheckboxTypeOptionPB {
   #[pb(index = 1)]
-  pub is_selected: bool,
+  pub config: bool,
```

Comment:
Is config an appropriate name here? It's a boolean...

Reply:
Yeah I don't know, it's a dummy field that doesn't do anything.

Reply:
Agreed with @Xazin. What does a bool value named "config" signify? Adding a comment to indicate it's a dummy field or renaming it would be helpful.